File: examples/frames/bias_eme2000.py
# /// script
# dependencies = ["brahe", "numpy"]
# ///
"""
Get the EME2000 frame bias matrix and verify its relationship to rotation functions
"""

import brahe as bh

# Get the EME2000 frame bias matrix
B = bh.bias_eme2000()

print("EME2000 frame bias matrix:")
print(f"  [{B[0, 0]:13.10f}, {B[0, 1]:13.10f}, {B[0, 2]:13.10f}]")
print(f"  [{B[1, 0]:13.10f}, {B[1, 1]:13.10f}, {B[1, 2]:13.10f}]")
print(f"  [{B[2, 0]:13.10f}, {B[2, 1]:13.10f}, {B[2, 2]:13.10f}]\n")

File: examples/frames/bias_eme2000.rs
//! Get the EME2000 frame bias matrix and verify its relationship to rotation functions

use brahe as bh;

fn main() {
    // Get the EME2000 frame bias matrix
    let b = bh::bias_eme2000();

    println!("EME2000 frame bias matrix:");
    println!("  [{:13.10}, {:13.10}, {:13.10}]", b[(0, 0)], b[(0, 1)], b[(0, 2)]);
    println!("  [{:13.10}, {:13.10}, {:13.10}]", b[(1, 0)], b[(1, 1)], b[(1, 2)]);
    println!("  [{:13.10}, {:13.10}, {:13.10}]\n", b[(2, 0)], b[(2, 1)], b[(2, 2)]);
}

